.github/workflows/crawler.yml

```
@@ -51,8 +51,14 @@ jobs:
           name: latest-result
           path: latest.json
 
-  call-process-results-workflow:
+  call-build-viz-state-workflow:
     needs: [ crawl-network ]
+    uses: runziggurat/ziggurat-core/.github/workflows/build-viz-state.yml@main
+    with:
+      filter: "Ripple"
```

`"` can be omitted here:
```suggestion
      filter: Ripple
```